notebooks/post_DL3_analysis.ipynb

```diff
@@ -0,0 +1,2633 @@
+{
```

Comment:
<p>You should not use the figure range to plot the sed. Use the range used for the fit. Maybe even limit to the used range with events.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>I just thought of having a large enough energy range, irrespective of the choice of the energy_fit_edges, to show the SEDs in a 'broad energy range' as a better 'visual' than fixing the plot limits based on the energy_fit_edges or even flux points estimated.</p><p> </p><p>I do not have a strong opinion on this. Can you elaborate more on your point?</p>

Reply:
I agree that a good comparison can only be made while considering the instrument's constraints. But showing the spectrum of the best fit model of the data and the reference, I think, should be ok to understand the trend of the spectral model used in both cases.

I chose 10 GeV to 40TeV, because, on average, I had events in the DL3 files, between 10-15 GeV and 40-45 TeV. So for just showing the LST1 spectrum, I assume that it is ok to show at least the spectrum, even if we do or do not include the upper limit flux points at the thresholds, to show it in the range we have the 'selected gamma-like' events.

But yes, when making a comparison with other analyses' SEDs, we should only plot within the reasonable energy range.

But, maybe my reasoning is flawed.